recipes/rmm/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "rmm" %}
+{% set version = "21.12.00" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rapidsai/rmm/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 71802422a04f2e0a828f7b7ec77e5cd1d24361d6dc5bf60a534659e4c2e08a44
+  folder: rmm
+
+build:
+  number: 0
+  skip: true  # [(not linux) or (cuda_compiler_version !=  "11.2") or (aarch64) or (py < 37) or (py >= 39)]
+
+  script_env:
+    - INSTALL_PREFIX=${PREFIX}
+    - CUDA_HOME  # [ppc64le]
+
+  script:
+    - cd rmm/python
+    - {{ PYTHON }} -m pip install . --no-deps -vv
+
+  missing_dso_whitelist:
+    - '*/libcuda.*'  # [linux]
+  ignore_run_exports:
+    - spdlog
+    - librmm
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+  host:
+    - librmm ={{ version }}
+    - python
+    - pip
+    - setuptools
+    - cython >=0.29,<0.30
+    - spdlog >=1.8.5,<2
+  run:
+    - python
+    - numba >=0.49
+    - numpy
+    - {{ pin_compatible('cudatoolkit', max_pin='x', lower_bound='11.2') }} # cudatoolkit >=11.2,<12.0.0
+
+test:
+  imports:
+    - rmm
+
+about:
+  home: https://github.com/rapidsai/rmm
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: rmm/LICENSE
```

Comment:
```suggestion
  license_file: LICENSE
```

Reply:
@carterbox Why? the check was passed before, did you see any CI failure?? The source is in the `./rmm` directory...

Reply:
My mistake. I didn't see LICENSE explicitly listed in the logs, so I assumed it wasn't the correct path.